data/transactions/logic/opcodes.go

```diff
@@ -229,6 +231,12 @@ func only(m RunMode) OpDetails {
 	return d
 }
 
+func deprecated(msg string) OpDetails {
```

Comment:
dog barking: do we actually use this method?

Reply:
meanwhile, it doesn't complain about `opDeprecated`. I wonder why?

Reply:
probably it only bark at new added non-comment lines of code. just a sheer guess.

Reply:
Yeah, that's probably it. I got `opDeprecated` in under last PR, when we needed it, but hadn't done the rest of the support, so only the new support is complained about here, even though I removed the `opDeprecated` use.